pkg/channel/consolidated/reconciler/controller/controller.go

```diff
@@ -125,5 +147,24 @@ func NewController(
 		Handler:    controller.HandleAll(grCh),
 	})
 
+	podInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
```

Comment:
IMO this should live inside the prober logic itself (when listing the endpoints, it should just do the difference)